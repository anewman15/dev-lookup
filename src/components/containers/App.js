import SearchUser from '../presentational/SearchUser';

const App = () => (
  <div className="App">
    <SearchUser />
  </div>
);

export default App;
