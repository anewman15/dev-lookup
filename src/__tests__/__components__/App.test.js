import { unmountComponentAtNode } from 'react-dom';
import renderer from 'react-test-renderer';
import App from '../../components/containers/App';

let container = null;
beforeEach(() => {
  container = document.createElement('div');
  document.body.appendChild(container);
});

afterEach(() => {
  unmountComponentAtNode(container);
  container.remove();
  container = null;
});

describe('App', () => {
  it('renders the App to the dom', () => {
    const tree = renderer
      .create(<App />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
