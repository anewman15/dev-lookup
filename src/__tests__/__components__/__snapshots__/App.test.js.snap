// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders the App to the dom 1`] = `
Array [
  <nav
    className="navBar"
  >
    <div
      className="container"
    >
      <div
        className="flexBetween"
      >
        <div
          className="flexBetween navLeft"
        >
          <a
            className="navLink"
            href="/"
            onClick={[Function]}
          >
            <img
              alt="dev-lookup-logo"
              className="logo"
              src="/logo.png"
            />
          </a>
        </div>
        <div
          className="flexBetween navRight"
        >
          <a
            className="navLink"
            href="/"
            onClick={[Function]}
          >
            Home
          </a>
          <a
            className="navLink"
            href="https://github.com/anewman15?tab=repositories"
          >
            anewman15
          </a>
        </div>
      </div>
    </div>
  </nav>,
  <div
    className="Home"
  >
    <div>
      <div
        className="container"
      >
        <div
          className="textCenter"
        >
          <h1>
            Look up a username on GitHub
          </h1>
          <form
            onSubmit={[Function]}
          >
            <input
              onChange={[Function]}
              placeholder="anewman15"
              type="text"
              value="anewman15"
            />
            <button
              type="submit"
            >
              Search
            </button>
          </form>
        </div>
        <hr />
      </div>
      <div
        className="container"
      >
        <h2
          className="textCenter "
        >
          Search for anewman15 returned 28 repositories
        </h2>
        <div
          className="container"
        >
          <div
            className="flexStart"
          >
            <h3>
              Filter by skills: 
            </h3>
            <form
              onSubmit={[Function]}
            >
              <input
                onChange={[Function]}
                placeholder="Ruby"
                type="text"
                value=""
              />
              <button
                type="submit"
              >
                Filter
              </button>
            </form>
          </div>
          
        </div>
        <div>
          <div
            className="repo"
          >
            <div
              className="flexBetween"
            >
              <div
                className="flexBetween"
              >
                <p
                  className="paragraph"
                >
                  1
                </p>
                <p
                  className="paragraph"
                >
                  Ruby
                </p>
              </div>
              <div
                className="flexStart"
              >
                <p
                  className="paragraph"
                >
                  Stars: 7
                </p>
                <p
                  className="paragraph"
                >
                  Last commited: 2021-02-26T15:07:02Z
                </p>
              </div>
            </div>
            <div
              className="repoUrl"
            >
              <a
                className="repoLink"
                href="/repo/anewman15/activize"
                onClick={[Function]}
              >
                https://github.com/anewman15/activize
              </a>
            </div>
          </div>
          <div
            className="repo"
          >
            <div
              className="flexBetween"
            >
              <div
                className="flexBetween"
              >
                <p
                  className="paragraph"
                >
                  2
                </p>
                <p
                  className="paragraph"
                />
              </div>
              <div
                className="flexStart"
              >
                <p
                  className="paragraph"
                >
                  Stars: 4
                </p>
                <p
                  className="paragraph"
                >
                  Last commited: 2021-02-19T12:01:44Z
                </p>
              </div>
            </div>
            <div
              className="repoUrl"
            >
              <a
                className="repoLink"
                href="/repo/anewman15/anewman15"
                onClick={[Function]}
              >
                https://github.com/anewman15/anewman15
              </a>
            </div>
          </div>
          <div
            className="repo"
          >
            <div
              className="flexBetween"
            >
              <div
                className="flexBetween"
              >
                <p
                  className="paragraph"
                >
                  3
                </p>
                <p
                  className="paragraph"
                >
                  CSS
                </p>
              </div>
              <div
                className="flexStart"
              >
                <p
                  className="paragraph"
                >
                  Stars: 13
                </p>
                <p
                  className="paragraph"
                >
                  Last commited: 2021-01-29T07:18:05Z
                </p>
              </div>
            </div>
            <div
              className="repoUrl"
            >
              <a
                className="repoLink"
                href="/repo/anewman15/backgrounds-and-gradients"
                onClick={[Function]}
              >
                https://github.com/anewman15/backgrounds-and-gradients
              </a>
            </div>
          </div>
          <div
            className="repo"
          >
            <div
              className="flexBetween"
            >
              <div
                className="flexBetween"
              >
                <p
                  className="paragraph"
                >
                  4
                </p>
                <p
                  className="paragraph"
                >
                  JavaScript
                </p>
              </div>
              <div
                className="flexStart"
              >
                <p
                  className="paragraph"
                >
                  Stars: 2
                </p>
                <p
                  className="paragraph"
                >
                  Last commited: 2021-03-05T09:51:35Z
                </p>
              </div>
            </div>
            <div
              className="repoUrl"
            >
              <a
                className="repoLink"
                href="/repo/anewman15/bookstore-react"
                onClick={[Function]}
              >
                https://github.com/anewman15/bookstore-react
              </a>
            </div>
          </div>
          <div
            className="repo"
          >
            <div
              className="flexBetween"
            >
              <div
                className="flexBetween"
              >
                <p
                  className="paragraph"
                >
                  5
                </p>
                <p
                  className="paragraph"
                >
                  CSS
                </p>
              </div>
              <div
                className="flexStart"
              >
                <p
                  className="paragraph"
                >
                  Stars: 8
                </p>
                <p
                  className="paragraph"
                >
                  Last commited: 2021-01-29T07:18:02Z
                </p>
              </div>
            </div>
            <div
              className="repoUrl"
            >
              <a
                className="repoLink"
                href="/repo/anewman15/bootstrap-nearby-groceries"
                onClick={[Function]}
              >
                https://github.com/anewman15/bootstrap-nearby-groceries
              </a>
            </div>
          </div>
          <div
            className="repo"
          >
            <div
              className="flexBetween"
            >
              <div
                className="flexBetween"
              >
                <p
                  className="paragraph"
                >
                  6
                </p>
                <p
                  className="paragraph"
                >
                  HTML
                </p>
              </div>
              <div
                className="flexStart"
              >
                <p
                  className="paragraph"
                >
                  Stars: 9
                </p>
                <p
                  className="paragraph"
                >
                  Last commited: 2021-01-29T07:18:03Z
                </p>
              </div>
            </div>
            <div
              className="repoUrl"
            >
              <a
                className="repoLink"
                href="/repo/anewman15/bootstrap-newsweek-magazine"
                onClick={[Function]}
              >
                https://github.com/anewman15/bootstrap-newsweek-magazine
              </a>
            </div>
          </div>
          <div
            className="repo"
          >
            <div
              className="flexBetween"
            >
              <div
                className="flexBetween"
              >
                <p
                  className="paragraph"
                >
                  7
                </p>
                <p
                  className="paragraph"
                >
                  Ruby
                </p>
              </div>
              <div
                className="flexStart"
              >
                <p
                  className="paragraph"
                >
                  Stars: 7
                </p>
                <p
                  className="paragraph"
                >
                  Last commited: 2021-01-29T18:36:59Z
                </p>
              </div>
            </div>
            <div
              className="repoUrl"
            >
              <a
                className="repoLink"
                href="/repo/anewman15/bubble-sort"
                onClick={[Function]}
              >
                https://github.com/anewman15/bubble-sort
              </a>
            </div>
          </div>
          <div
            className="repo"
          >
            <div
              className="flexBetween"
            >
              <div
                className="flexBetween"
              >
                <p
                  className="paragraph"
                >
                  8
                </p>
                <p
                  className="paragraph"
                >
                  JavaScript
                </p>
              </div>
              <div
                className="flexStart"
              >
                <p
                  className="paragraph"
                >
                  Stars: 0
                </p>
                <p
                  className="paragraph"
                >
                  Last commited: 2021-03-06T15:57:57Z
                </p>
              </div>
            </div>
            <div
              className="repoUrl"
            >
              <a
                className="repoLink"
                href="/repo/anewman15/dev-lookup"
                onClick={[Function]}
              >
                https://github.com/anewman15/dev-lookup
              </a>
            </div>
          </div>
          <div
            className="repo"
          >
            <div
              className="flexBetween"
            >
              <div
                className="flexBetween"
              >
                <p
                  className="paragraph"
                >
                  9
                </p>
                <p
                  className="paragraph"
                >
                  JavaScript
                </p>
              </div>
              <div
                className="flexStart"
              >
                <p
                  className="paragraph"
                >
                  Stars: 4
                </p>
                <p
                  className="paragraph"
                >
                  Last commited: 2021-01-29T18:36:33Z
                </p>
              </div>
            </div>
            <div
              className="repoUrl"
            >
              <a
                className="repoLink"
                href="/repo/anewman15/Library"
                onClick={[Function]}
              >
                https://github.com/anewman15/Library
              </a>
            </div>
          </div>
          <div
            className="repo"
          >
            <div
              className="flexBetween"
            >
              <div
                className="flexBetween"
              >
                <p
                  className="paragraph"
                >
                  10
                </p>
                <p
                  className="paragraph"
                >
                  Ruby
                </p>
              </div>
              <div
                className="flexStart"
              >
                <p
                  className="paragraph"
                >
                  Stars: 6
                </p>
                <p
                  className="paragraph"
                >
                  Last commited: 2021-01-29T18:36:40Z
                </p>
              </div>
            </div>
            <div
              className="repoUrl"
            >
              <a
                className="repoLink"
                href="/repo/anewman15/members-only"
                onClick={[Function]}
              >
                https://github.com/anewman15/members-only
              </a>
            </div>
          </div>
          <div
            className="repo"
          >
            <div
              className="flexBetween"
            >
              <div
                className="flexBetween"
              >
                <p
                  className="paragraph"
                >
                  11
                </p>
                <p
                  className="paragraph"
                >
                  Ruby
                </p>
              </div>
              <div
                className="flexStart"
              >
                <p
                  className="paragraph"
                >
                  Stars: 6
                </p>
                <p
                  className="paragraph"
                >
                  Last commited: 2021-01-29T18:36:47Z
                </p>
              </div>
            </div>
            <div
              className="repoUrl"
            >
              <a
                className="repoLink"
                href="/repo/anewman15/micro-reddit"
                onClick={[Function]}
              >
                https://github.com/anewman15/micro-reddit
              </a>
            </div>
          </div>
          <div
            className="repo"
          >
            <div
              className="flexBetween"
            >
              <div
                className="flexBetween"
              >
                <p
                  className="paragraph"
                >
                  12
                </p>
                <p
                  className="paragraph"
                >
                  HTML
                </p>
              </div>
              <div
                className="flexStart"
              >
                <p
                  className="paragraph"
                >
                  Stars: 14
                </p>
                <p
                  className="paragraph"
                >
                  Last commited: 2021-01-29T07:18:07Z
                </p>
              </div>
            </div>
            <div
              className="repoUrl"
            >
              <a
                className="repoLink"
                href="/repo/anewman15/nyt-article-clone"
                onClick={[Function]}
              >
                https://github.com/anewman15/nyt-article-clone
              </a>
            </div>
          </div>
          <div
            className="repo"
          >
            <div
              className="flexBetween"
            >
              <div
                className="flexBetween"
              >
                <p
                  className="paragraph"
                >
                  13
                </p>
                <p
                  className="paragraph"
                >
                  Ruby
                </p>
              </div>
              <div
                className="flexStart"
              >
                <p
                  className="paragraph"
                >
                  Stars: 7
                </p>
                <p
                  className="paragraph"
                >
                  Last commited: 2021-02-26T15:07:12Z
                </p>
              </div>
            </div>
            <div
              className="repoUrl"
            >
              <a
                className="repoLink"
                href="/repo/anewman15/nytimes-articles-scraper"
                onClick={[Function]}
              >
                https://github.com/anewman15/nytimes-articles-scraper
              </a>
            </div>
          </div>
          <div
            className="repo"
          >
            <div
              className="flexBetween"
            >
              <div
                className="flexBetween"
              >
                <p
                  className="paragraph"
                >
                  14
                </p>
                <p
                  className="paragraph"
                >
                  JavaScript
                </p>
              </div>
              <div
                className="flexStart"
              >
                <p
                  className="paragraph"
                >
                  Stars: 2
                </p>
                <p
                  className="paragraph"
                >
                  Last commited: 2021-02-11T07:00:59Z
                </p>
              </div>
            </div>
            <div
              className="repoUrl"
            >
              <a
                className="repoLink"
                href="/repo/anewman15/portfolio-react"
                onClick={[Function]}
              >
                https://github.com/anewman15/portfolio-react
              </a>
            </div>
          </div>
          <div
            className="repo"
          >
            <div
              className="flexBetween"
            >
              <div
                className="flexBetween"
              >
                <p
                  className="paragraph"
                >
                  15
                </p>
                <p
                  className="paragraph"
                >
                  Ruby
                </p>
              </div>
              <div
                className="flexStart"
              >
                <p
                  className="paragraph"
                >
                  Stars: 7
                </p>
                <p
                  className="paragraph"
                >
                  Last commited: 2021-02-26T15:07:09Z
                </p>
              </div>
            </div>
            <div
              className="repoUrl"
            >
              <a
                className="repoLink"
                href="/repo/anewman15/private-events"
                onClick={[Function]}
              >
                https://github.com/anewman15/private-events
              </a>
            </div>
          </div>
          <div
            className="repo"
          >
            <div
              className="flexBetween"
            >
              <div
                className="flexBetween"
              >
                <p
                  className="paragraph"
                >
                  16
                </p>
                <p
                  className="paragraph"
                >
                  Ruby
                </p>
              </div>
              <div
                className="flexStart"
              >
                <p
                  className="paragraph"
                >
                  Stars: 5
                </p>
                <p
                  className="paragraph"
                >
                  Last commited: 2021-01-29T18:36:44Z
                </p>
              </div>
            </div>
            <div
              className="repoUrl"
            >
              <a
                className="repoLink"
                href="/repo/anewman15/re-former"
                onClick={[Function]}
              >
                https://github.com/anewman15/re-former
              </a>
            </div>
          </div>
          <div
            className="repo"
          >
            <div
              className="flexBetween"
            >
              <div
                className="flexBetween"
              >
                <p
                  className="paragraph"
                >
                  17
                </p>
                <p
                  className="paragraph"
                >
                  JavaScript
                </p>
              </div>
              <div
                className="flexStart"
              >
                <p
                  className="paragraph"
                >
                  Stars: 1
                </p>
                <p
                  className="paragraph"
                >
                  Last commited: 2021-02-24T17:11:26Z
                </p>
              </div>
            </div>
            <div
              className="repoUrl"
            >
              <a
                className="repoLink"
                href="/repo/anewman15/react-calculator"
                onClick={[Function]}
              >
                https://github.com/anewman15/react-calculator
              </a>
            </div>
          </div>
          <div
            className="repo"
          >
            <div
              className="flexBetween"
            >
              <div
                className="flexBetween"
              >
                <p
                  className="paragraph"
                >
                  18
                </p>
                <p
                  className="paragraph"
                >
                  JavaScript
                </p>
              </div>
              <div
                className="flexStart"
              >
                <p
                  className="paragraph"
                >
                  Stars: 4
                </p>
                <p
                  className="paragraph"
                >
                  Last commited: 2021-01-29T18:36:31Z
                </p>
              </div>
            </div>
            <div
              className="repoUrl"
            >
              <a
                className="repoLink"
                href="/repo/anewman15/restaurant-page"
                onClick={[Function]}
              >
                https://github.com/anewman15/restaurant-page
              </a>
            </div>
          </div>
          <div
            className="repo"
          >
            <div
              className="flexBetween"
            >
              <div
                className="flexBetween"
              >
                <p
                  className="paragraph"
                >
                  19
                </p>
                <p
                  className="paragraph"
                >
                  Ruby
                </p>
              </div>
              <div
                className="flexStart"
              >
                <p
                  className="paragraph"
                >
                  Stars: 7
                </p>
                <p
                  className="paragraph"
                >
                  Last commited: 2021-02-26T15:07:06Z
                </p>
              </div>
            </div>
            <div
              className="repoUrl"
            >
              <a
                className="repoLink"
                href="/repo/anewman15/ror-social-scaffold"
                onClick={[Function]}
              >
                https://github.com/anewman15/ror-social-scaffold
              </a>
            </div>
          </div>
          <div
            className="repo"
          >
            <div
              className="flexBetween"
            >
              <div
                className="flexBetween"
              >
                <p
                  className="paragraph"
                >
                  20
                </p>
                <p
                  className="paragraph"
                >
                  Ruby
                </p>
              </div>
              <div
                className="flexStart"
              >
                <p
                  className="paragraph"
                >
                  Stars: 5
                </p>
                <p
                  className="paragraph"
                >
                  Last commited: 2021-01-29T18:36:56Z
                </p>
              </div>
            </div>
            <div
              className="repoUrl"
            >
              <a
                className="repoLink"
                href="/repo/anewman15/ruby-enumerables"
                onClick={[Function]}
              >
                https://github.com/anewman15/ruby-enumerables
              </a>
            </div>
          </div>
          <div
            className="repo"
          >
            <div
              className="flexBetween"
            >
              <div
                className="flexBetween"
              >
                <p
                  className="paragraph"
                >
                  21
                </p>
                <p
                  className="paragraph"
                >
                  Ruby
                </p>
              </div>
              <div
                className="flexStart"
              >
                <p
                  className="paragraph"
                >
                  Stars: 5
                </p>
                <p
                  className="paragraph"
                >
                  Last commited: 2021-01-29T18:36:54Z
                </p>
              </div>
            </div>
            <div
              className="repoUrl"
            >
              <a
                className="repoLink"
                href="/repo/anewman15/ruby-tic-tac-toe"
                onClick={[Function]}
              >
                https://github.com/anewman15/ruby-tic-tac-toe
              </a>
            </div>
          </div>
          <div
            className="repo"
          >
            <div
              className="flexBetween"
            >
              <div
                className="flexBetween"
              >
                <p
                  className="paragraph"
                >
                  22
                </p>
                <p
                  className="paragraph"
                >
                  HTML
                </p>
              </div>
              <div
                className="flexStart"
              >
                <p
                  className="paragraph"
                >
                  Stars: 12
                </p>
                <p
                  className="paragraph"
                >
                  Last commited: 2021-01-29T07:18:04Z
                </p>
              </div>
            </div>
            <div
              className="repoUrl"
            >
              <a
                className="repoLink"
                href="/repo/anewman15/smashing-magazine-heatmap"
                onClick={[Function]}
              >
                https://github.com/anewman15/smashing-magazine-heatmap
              </a>
            </div>
          </div>
          <div
            className="repo"
          >
            <div
              className="flexBetween"
            >
              <div
                className="flexBetween"
              >
                <p
                  className="paragraph"
                >
                  23
                </p>
                <p
                  className="paragraph"
                >
                  JavaScript
                </p>
              </div>
              <div
                className="flexStart"
              >
                <p
                  className="paragraph"
                >
                  Stars: 4
                </p>
                <p
                  className="paragraph"
                >
                  Last commited: 2021-01-29T18:36:23Z
                </p>
              </div>
            </div>
            <div
              className="repoUrl"
            >
              <a
                className="repoLink"
                href="/repo/anewman15/spaceship-shooter"
                onClick={[Function]}
              >
                https://github.com/anewman15/spaceship-shooter
              </a>
            </div>
          </div>
          <div
            className="repo"
          >
            <div
              className="flexBetween"
            >
              <div
                className="flexBetween"
              >
                <p
                  className="paragraph"
                >
                  24
                </p>
                <p
                  className="paragraph"
                >
                  TSQL
                </p>
              </div>
              <div
                className="flexStart"
              >
                <p
                  className="paragraph"
                >
                  Stars: 5
                </p>
                <p
                  className="paragraph"
                >
                  Last commited: 2021-01-29T18:36:49Z
                </p>
              </div>
            </div>
            <div
              className="repoUrl"
            >
              <a
                className="repoLink"
                href="/repo/anewman15/sql-from-sqlzoo"
                onClick={[Function]}
              >
                https://github.com/anewman15/sql-from-sqlzoo
              </a>
            </div>
          </div>
          <div
            className="repo"
          >
            <div
              className="flexBetween"
            >
              <div
                className="flexBetween"
              >
                <p
                  className="paragraph"
                >
                  25
                </p>
                <p
                  className="paragraph"
                >
                  HTML
                </p>
              </div>
              <div
                className="flexStart"
              >
                <p
                  className="paragraph"
                >
                  Stars: 2
                </p>
                <p
                  className="paragraph"
                >
                  Last commited: 2021-02-11T13:55:45Z
                </p>
              </div>
            </div>
            <div
              className="repoUrl"
            >
              <a
                className="repoLink"
                href="/repo/anewman15/tic-tac-toe-react"
                onClick={[Function]}
              >
                https://github.com/anewman15/tic-tac-toe-react
              </a>
            </div>
          </div>
          <div
            className="repo"
          >
            <div
              className="flexBetween"
            >
              <div
                className="flexBetween"
              >
                <p
                  className="paragraph"
                >
                  26
                </p>
                <p
                  className="paragraph"
                >
                  JavaScript
                </p>
              </div>
              <div
                className="flexStart"
              >
                <p
                  className="paragraph"
                >
                  Stars: 6
                </p>
                <p
                  className="paragraph"
                >
                  Last commited: 2021-02-26T15:07:16Z
                </p>
              </div>
            </div>
            <div
              className="repoUrl"
            >
              <a
                className="repoLink"
                href="/repo/anewman15/to-dos-app"
                onClick={[Function]}
              >
                https://github.com/anewman15/to-dos-app
              </a>
            </div>
          </div>
          <div
            className="repo"
          >
            <div
              className="flexBetween"
            >
              <div
                className="flexBetween"
              >
                <p
                  className="paragraph"
                >
                  27
                </p>
                <p
                  className="paragraph"
                >
                  Ruby
                </p>
              </div>
              <div
                className="flexStart"
              >
                <p
                  className="paragraph"
                >
                  Stars: 6
                </p>
                <p
                  className="paragraph"
                >
                  Last commited: 2021-01-29T18:36:43Z
                </p>
              </div>
            </div>
            <div
              className="repoUrl"
            >
              <a
                className="repoLink"
                href="/repo/anewman15/twitter-clone"
                onClick={[Function]}
              >
                https://github.com/anewman15/twitter-clone
              </a>
            </div>
          </div>
          <div
            className="repo"
          >
            <div
              className="flexBetween"
            >
              <div
                className="flexBetween"
              >
                <p
                  className="paragraph"
                >
                  28
                </p>
                <p
                  className="paragraph"
                >
                  JavaScript
                </p>
              </div>
              <div
                className="flexStart"
              >
                <p
                  className="paragraph"
                >
                  Stars: 5
                </p>
                <p
                  className="paragraph"
                >
                  Last commited: 2021-02-26T15:07:25Z
                </p>
              </div>
            </div>
            <div
              className="repoUrl"
            >
              <a
                className="repoLink"
                href="/repo/anewman15/weathr"
                onClick={[Function]}
              >
                https://github.com/anewman15/weathr
              </a>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
]
`;
